### PR TITLE
Adds SFG experiment branches to doubleclick-a4a-config.js.

### PR DESCRIPTION
**Pull request by glevitzky**
The new control and treatment branches correspond to exp=a4a:5 and exp=a4a:6, respectively.

### CODE REVIEW

File: ads/google/a4a/traffic-experiments.js [hunk 1]
```diff
@@ -90,8 +91,8 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
   const isSetFromUrl = maybeSetExperimentFromUrl(win, element,
       experimentName, externalBranches.control,
       externalBranches.experiment, delayedExternalBranches.control,
-      delayedExternalBranches.experiment,
-      MANUAL_EXPERIMENT_ID);
+      delayedExternalBranches.experiment, sfgInternalBranches.control,
+      sfgInternalBranches.experiment, MANUAL_EXPERIMENT_ID);
   const experimentInfoMap = {};
   const branches = [
     internalBranches.control,
```
Comment (jonkeller) [inline ads/google/a4a/traffic-experiments.js]: I suspect you'll want to add sfgInternalBranches.experiment to the "const selected" expression on line 132 below.

Comment (jonkeller) [inline ads/google/a4a/traffic-experiments.js]: ...actually, based on your code comment on line 210, I am probably wrong about that.

File: ads/google/a4a/traffic-experiments.js [hunk 2]
```diff
@@ -201,10 +206,13 @@ function maybeSetExperimentFromUrl(win, element, experimentName,
     '2': treatmentBranchId,
     '3': delayedControlId,
     '4': delayedTreatmentBrandId,
+    '5': sfgControlId,
+    '6': sfgTreatmentId,
   };
   if (argMapping.hasOwnProperty(arg)) {
     forceExperimentBranch(win, experimentName, argMapping[arg]);
-    return true;
+    // Neither branch of SFG is eligible for A4A, unless we're an AdSense ad.
```
Comment (keithwrightbos) [inline ads/google/a4a/traffic-experiments.js]: Why do we need this?  Why can we not return true?  That is only indicative of whether we successfully selected into an experiment?

Comment (glevitzky) [inline ads/google/a4a/traffic-experiments.js]: Misunderstood the purpose of this function's return statement. Everything works without this change, anyway.

**Pull request merged**